packages/mdc-textfield/adapter.js

```diff
@@ -43,7 +43,7 @@ let NativeInputType;
  *   helperText: (!MDCTextFieldHelperTextFoundation|undefined),
  *   icon: (!MDCTextFieldIconFoundation|undefined),
  *   label: (!MDCFloatingLabelFoundation|undefined),
- *   outline: (!MDCTextFieldOutlineFoundation|undefined)
+ *   outline: (!MDCNotchedOutlineFoundation|undefined)
```

Comment:
You dont need lineRipple, label, or outline anymore. `MDCTextFieldFoundation` only needs helperText and icon